Makefile

```
@@ -85,6 +85,17 @@ golangci-lint:
 	curl -sSfL https://raw.githubusercontent.com/golangci/golangci-lint/master/install.sh | sh -s -- -b $(shell dirname $(GOLANGCI_LINT)) v1.37.1 ;\
 	}
 
+.PHONY: apidiff
+apidiff: go-apidiff ## Run the go-apidiff to verify any API differences compared with origin/master
+	git fetch origin
+	$(GO_APIDIFF) origin/master --compare-imports --print-compatible --repo-path=.
```

I'd rather leave out git-related commands from targets, since local repo configuration can vary.
```suggestion
	$(GO_APIDIFF) master --compare-imports --print-compatible --repo-path=.
```